msal/src/test/java/com/microsoft/identity/client/AuthenticationResultTest.java

```diff
@@ -0,0 +1,93 @@
+package com.microsoft.identity.client;
```

Comment:
License info?

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1063/commits/353f46523435797d9079cbf600a795d288eda4ed